packages/api/src/utils/nonce_lab/create_survey_request.rs

```diff
@@ -81,24 +81,62 @@ impl From<PanelV2> for NonceLabQuota {
             ..
         }: PanelV2,
     ) -> Self {
-        let mut age = None;
-        let mut gender = None;
-        let mut region = None;
-        let mut salary = None;
+        let mut age: Vec<NonceLabAge> = vec![];
+        let mut gender: Vec<u8> = vec![];
+        let mut region: Vec<RegionCode> = vec![];
+        let mut salary: Vec<SalaryTier> = vec![];
 
         for attribute in attributes.clone() {
             match attribute {
                 Attribute::Age(age_v3) => {
-                    age = Some(age_v3);
+                    match age_v3 {
+                        AgeV3::Specific(v) => {
+                            let a = NonceLabAge::Specific(v);
+                            age.push(a);
+                        }
+                        AgeV3::Range {
+                            inclusive_min,
+                            inclusive_max,
+                        } => {
+                            let a = NonceLabAge::Range {
+                                inclusive_min,
+                                inclusive_max,
+                            };
+                            age.push(a);
+                        }
+                        AgeV3::None => {}
+                    };
                 }
                 Attribute::Gender(gender_v2) => {
-                    gender = Some(gender_v2);
+                    let g = gender_v2 as u8;
+                    gender.push(g);
                 }
                 Attribute::Region(region_v2) => {
-                    region = Some(region_v2);
+                    let r = match region_v2 {
+                        RegionV2::Seoul => 02 as RegionCode,
+                        RegionV2::Busan => 051 as RegionCode,
+                        RegionV2::Daegu => 053 as RegionCode,
```

Comment:
implement `From<RegionV2> for RegionCode`
then use `let r = region_v2.into();`

Reply:
I fixed it.